hs_swat_modelinstance/models.py

```diff
@@ -493,10 +497,10 @@ def update(self, metadata, user):
                         self.update_non_repeatable_element(element_name, metadata,
                                                            element_property_name)
 
-    def get_xml(self, pretty_print=True, include_format_elements=True):
+    def get_xml_legacy(self, pretty_print=True, include_format_elements=True):
```

Comment:
Does _legacy mean you're planning on replacing this later?

Reply:
https://github.com/hydroshare/hydroshare/pull/3928#discussion_r530439375